doc/conf.py

```diff
@@ -125,7 +125,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+html_static_path = []
```

Comment:
This seems unrelated to the Python 3 update at first sight. I am not sure _static was doing anything, though?

Reply:
Not really related to Python 3 but I also tried to fix the tests and I got an error of a non-existing '_static' directory in 'src'. As we don't use this directory we should also remove it from the configuration.